Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,479 @@
+    , sosNATDestinationPort = dnpt // NAT'ed destination port
+    , sosSourcePortSPT = spt // Source port
+    , sosNATSourcePort = snpt // NAT'ed source port
+    , sosBladeID = bid // Blade ID
+    , sosAuditID = auditId // Audit ID of the Configuration Auditing event.
+    , sosAuditTransactionID = tranxId // Transaction ID of the entries pertaining to the same transaction or batch.
+    , sosAuditUserPrivileges = userMode // User Mode. Displays the privileges assigned to a user.
+    , sosAuditGroupName = grpName // Labels a group of data usually associated with a table (ex: VPN Policy list, Syslog Server list, etc.)
+    , sosAuditGroupIndex = grpIndex // Labels the instance of the data group (ex: a VPN policy name, Syslog server entry in the list, etc.)
+    , sosAuditOldValue = oldValue // The previous value associated with the auditing event.
+    , sosAuditNewValue = newValue // The new value associated with the auditing event.
+    , sosAuditTime = auditTime // The timestamp indicates the time configuration was changed.
+    , sosAuditPath = auditPath // The logical location of the configuration setting.
+    , sosUUID = uuid
+    , sosHttpRequestMethod = HttpRequestMethod
+    , sosFileName = FileName
+    , DvcOriginalAction = fw_action
+  | extend AdditionalFields = bag_pack(
+      "ReceivedPackets_Field", sosReceivedPackets_Field
+    , "ReceivedPackets", sosReceivedPackets
+    , "SentPackets_Field", sosSentPackets_Field
+    , "SentPackets", sosSentPackets
+    , "ConnectionDuration", sosConnectionDuration
+    , "User", sosUser
+    , "AppRulePolicyId", sosAppRulePolicyId
+    , "AppRulePolicyName", sosAppRulePolicyName
+    , "AppRuleService", sosAppRuleService
+    , "AppRuleType", sosAppRuleType
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "SourceIPv6Address", sosSourceIPv6Address
+    , "DestinationIPv6Address", sosDestinationIPv6Address
+    , "ServiceName", sosServiceName
+    , "AppFullString", sosAppFullString
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "DestinationPortDPT", sosDestinationPortDPT
+    , "NATDestinationPort", sosNATDestinationPort
+    , "SourcePortSPT", sosSourcePortSPT
+    , "NATSourcePort", sosNATSourcePort
+    , "BladeID", sosBladeID
+    , "AuditID", sosAuditID
+    , "AuditTransactionID", sosAuditTransactionID
+    , "AuditUserPrivileges", sosAuditUserPrivileges
+    , "AuditGroupName", sosAuditGroupName
+    , "AuditGroupIndex", sosAuditGroupIndex
+    , "AuditOldValue", sosAuditOldValue
+    , "AuditNewValue", sosAuditNewValue

```

Comment:
@jaimeesc these seems like Audit event fields. I don't think these would contain any value for NetworkSession event.
FYI. We have audit schema for such events: https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-network

Reply:
I'll remove these.